neonvm-kernel/linux-config-aarch64-6.12.24

```diff
@@ -2097,10 +2113,9 @@ CONFIG_BCMA_POSSIBLE=y
 #
 CONFIG_APERTURE_HELPERS=y
 CONFIG_SCREEN_INFO=y
-CONFIG_VIDEO_CMDLINE=y
+CONFIG_VIDEO=y
```

Comment:
Looks like this is probably required by `CONFIG_FB=y` (framebuffer support). Maybe worth removing as a follow-up.